gluster-exporter/metric_volume.go

```diff
@@ -205,7 +205,15 @@ func profileInfo() error {
 	if err != nil {
 		return err
 	}
+	volOption := glusterutils.CountFOPHitsGD1
+	if glusterConfig.GlusterMgmt == glusterutils.MgmtGlusterd2 {
+		volOption = glusterutils.CountFOPHitsGD2
+	}
 	for _, volume := range volumes {
+		if value, exists := volume.Options[volOption]; exists && value == "off" {
```

Comment:
If volinfo.Options doesn't contain value that also means do not collect this metric. Only possible when user does option reset once exporter starts.

`!exists || value == "off"`

Reply:
Ack. Will change the logic accordingly.

Reply:
So in what situation we need to enable profiling for the volumes?